*** linux/fermisurfer.c	Thu Apr 30 21:53:59 2015
--- mac/fermisurfer.c	Fri May  8 19:49:30 2015
***************
*** 24,31 ****
  #include <stdlib.h>
  #include <stdio.h>
  #include <math.h>
! #include <GL/glut.h>
! #include <omp.h>
  /**
  * Input variables
  */
--- 24,30 ----
  #include <stdlib.h>
  #include <stdio.h>
  #include <math.h>
! #include <GLUT/glut.h>
  /**
  * Input variables
  */
***************
*** 2078,2084 ****
    max_and_min_bz();
    /**/
  #pragma omp parallel 
-   printf("Threads : %d \n", omp_get_num_threads());
    /**/
    printf("\n#####  First Brillouin zone mode  ##### \n\n");
    query = 1;
--- 2077,2082 ----
